### PR TITLE
Fix the reject/accept call buttons in canary (mk2)

### PR DESCRIPTION
**Pull request by dbkr**
Fixes https://github.com/vector-im/riot-web/issues/6081 by making
the accept/reject buttons AccessibleButtons which they should be
anyway (presumably the role=button makes chrome do the right thing
with the events). Also swallow the onClick event otherwise that
propagates out to the room header and causes it to expand/collapse.

### CODE REVIEW

File: src/components/views/voip/IncomingCallBox.js [hunk 1]
```diff
@@ -18,6 +19,7 @@ import PropTypes from 'prop-types';
 import MatrixClientPeg from '../../../MatrixClientPeg';
 import dis from '../../../dispatcher';
 import { _t } from '../../../languageHandler';
+import AccessibleButton from '../elements/AccessibleButton';
```
Comment (lukebarnard1) [inline src/components/views/voip/IncomingCallBox.js]: Should this not be taken via `getComponent` ?

Comment (dbkr) [reply src/components/views/voip/IncomingCallBox.js]: mm, yes, it should. For some reason we've used import directly elsewhere in the codebase.

File: src/components/views/voip/IncomingCallBox.js [hunk 2]
```diff
@@ -59,6 +62,7 @@ module.exports = React.createClass({
             }
         }
 
+        const AccessibleButton = sdk.getComponent('elements.AccessibleButton');
```
Comment (lukebarnard1) [inline src/components/views/voip/IncomingCallBox.js]: sdk has not been imported

Comment (dbkr) [reply src/components/views/voip/IncomingCallBox.js]: oops, good catch. Now actually tested too...

**Comment by lukebarnard1** [comment]
There's a missing semi-colon on the import

**Pull request merged**